actpool/actpool.go

```diff
@@ -392,6 +376,16 @@ func (ap *actPool) enqueueAction(sender string, act action.SealedEnvelope, hash
 	}
 	ap.allActions[hash] = act
 
+	//add actions to destination map
+	desAddress, ok := act.Destination()
+	if ok && !strings.EqualFold(sender, desAddress) {
+		dst := ap.accountDesActs[desAddress]
+		if dst == nil {
+			ap.accountDesActs[desAddress] = make([]action.SealedEnvelope, 0)
+		}
```

Comment:
I think this is not necessary in golang. you can append item to nil. Please double check

Reply:
right,it's not necessary